mk-local-hackage-repo/default.nix

```diff
@@ -24,6 +24,12 @@ export expires="4000-01-01T00:00:00Z"
 ln -sf ${index} $out/01-index.tar.gz
 export index_md5=$(nix-hash --flat --type md5 ${index})
 export index_sha256=$(nix-hash --flat --type sha256 ${index})
+${ pkgs.lib.optionalString (index ? outputHash) ''
```

Comment:
What does it mean if the attribute is missing?

Reply:
The `index` can come from `extra-hackages` and I don't think we want to require an outputHash for those.  For instance the test for `extra-hackages` does not include one https://github.com/input-output-hk/haskell.nix/blob/58e5214901dc3e5220a6f5875b11521cfa8c1a35/test/extra-hackage/default.nix#L11

I have added a comment to the code.